test/ScalaTestResourcesFromLocalTargetTest.scala

```diff
@@ -0,0 +1,8 @@
+package scalarules.test
+
+class ScalaTestResourcesFromLocalTargetTest extends org.scalatest.FlatSpec {
+  "scala_test's resources when referencing local target" should "assert that local target is not placed in bazel-out, but rather next to the generated code" in {
+    assert(getClass.getResource("/bazel-out/darwin-fastbuild/bin/test/py_resource_binary") == null)
```

Comment:
If I'm correctly reading the test failure you linked to previously this assertion passed and only the next one failed.
What is its purpose?
Not saying you should remove it but I'm not 100% sure about it

Reply:
Huh. That _should_ never pass. I pushed up a branch with just that assert and it seems to fail. Not sure what you were looking at, but I'd be very curious to know how that happened.

Reply:
👍🏽 I was looking at the test before the line break so line 6 was the second assertion.